papyri/gen.py

```diff
@@ -1832,6 +1854,11 @@ def collect_api_docs(
             )
 
         collected = {k: v for k, v in collected.items() if k not in excluded}
+        if limit_to:
+            collected = {k: v for k, v in collected.items() if k in limit_to}
+            print("DEV: regenerating docs only for")
```

Comment:
I see the logging is setup already, any specific reason for using `print`?

Reply:
Good point.